src/DIRAC/tests/Utilities/testJobDefinitions.py

```diff
@@ -209,6 +209,22 @@ def helloWorldSSHBatch():
     return endOfAllJobs(J)
 
 
+def helloWorldARM():
+    """simple hello world job to DIRAC.ARM.ch"""
+
+    J = baseToAllJobs("helloWorldARM")
+    try:
+        J.setInputSandbox([find_all("exe-script.py", rootPath, "DIRAC/tests/Workflow")[0]])
+    except IndexError:
+        try:
+            J.setInputSandbox([find_all("exe-script.py", ".", "DIRAC/tests/Workflow")[0]])
+        except IndexError:  # we are in Jenkins
+            J.setInputSandbox([find_all("exe-script.py", os.environ["WORKSPACE"], "DIRAC/tests/Workflow")[0]])
+    J.setExecutable("exe-script.py", "", "helloWorld.log")
+    J.setDestination("DIRAC.ARM.ch")
```

Comment:
No need to set a tag? Such as:

```python
J.setTag(['ARM'])
```

Reply:
Not necessarily, since anyway it targets specifically a "DIRAC.ARM.ch" site. Of course such site would be able to match other jobs too, but for the "helloworld" job types that we do in DIRAC certif it is probably OK.